src/snowflake/cli/api/exceptions.py

```diff
@@ -94,3 +95,15 @@ def __init__(self, object_name: str):
         super().__init__(
             f"{object_name} defined with secrets but without external integration."
         )
+
+
+class FileTooLargeError(ClickException):
+    def __init__(self, path: Path, size_limit_in_kb: int):
+        super().__init__(
+            f"File {path} is too large (size limit: {size_limit_in_kb} KB)"
```

Comment:
Did we decide the limit? I think KB is too low, we should have MB.

One important thing, this value should be able to be overwritten by value from `config.toml`. We should not block people to read larger files. WDYT?

Reply:
1. Done
2. Discussed on daily